layers/+lang/asm/packages.el

```diff
@@ -56,3 +58,9 @@
 (defun asm/post-init-company ()
   (spacemacs|add-company-hook asm-mode)
   (spacemacs|add-company-hook nasm-mode))
+
+(defun asm/post-init-ggtags ()
+  (add-hook 'asm-mode-hook '(lambda () (ggtags-mode 1))))
```

Comment:
Why do you not use `spacemacs/ggtags-enable-eldoc` here instead of `add-hook`?
Same goes for most other places where `(add-hook ... ggtags-mode)` is used.